examples/ui/text_wrap_debug.rs

```diff
@@ -1,13 +1,41 @@
 //! This example demonstrates text wrapping and use of the `LineBreakOn` property.
 
+use argh::FromArgs;
 use bevy::prelude::*;
 use bevy::text::BreakLineOn;
 use bevy::winit::WinitSettings;
+use bevy_internal::window::WindowResolution;
+
+#[derive(FromArgs, Resource)]
+/// `text_wrap_debug` demonstrates text wrapping and use of the `LineBreakOn` property
+struct Args {
+    #[argh(option)]
+    /// window scale factor
+    scale_factor: Option<f64>,
```

Comment:
Why is one of these an Option but not the other?

Reply:
A default setting for window scale factor would override whatever scale factor the user or OS has set for the monitor.